atom/browser/native_window_mac.mm

```diff
@@ -334,9 +326,6 @@ void ViewDidMoveToSuperview(NSView* self, SEL _cmd) {
     // The window without titlebar is treated the same with frameless window.
     set_has_frame(false);
   }
-  if (!useStandardWindow || transparent() || !has_frame()) {
-    styleMask |= NSTexturedBackgroundWindowMask;
```

Comment:
This concerns me, deprecated or not we should keep this behavior until the API is completely removed.  Frameless / transparent windows depend on this mask under some conditions iirc from the Mojave White Line incident

Reply:
Do you know which conditions specifically? I'd prefer to try to work around those then keep this if that's a viable pathway i'd say

Reply:
@codebytere I think High Sierra and below 🤔 I can't recall / guarantee to know all the cases though.  Until it's removed from the SDK I'd prefer we kept it

Reply:
i can close this PR then :)